src/hooks/useSettingsUtilities.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  useDeskproAppClient,
+  useDeskproLatestAppContext,
+} from "@deskpro/app-sdk";
+import { IBookmark } from "../types/bookmarks";
+
+type Props = {
+  getBookmarks: () => IBookmark[];
+  getParentFolders: () => IBookmark[];
+  addBookmark: (bookmark: IBookmark) => void;
+  editBookmark: (bookmark: IBookmark) => void;
+  moveBookmark: (bookmarkId: string, newIndex: number) => void;
+  removeBookmark: (bookmarkId: string) => void;
+};
+
+export const useSettingsUtilities = (): Props | null => {
```

Comment:
Maybe we will do looks like this (because `props` it's __input__ parameters)

```typescript
type UseSettingsUtilities = () => {
  getBookmarks: () => IBookmark[];
  getParentFolders: () => IBookmark[];
  addBookmark: (bookmark: IBookmark) => void;
  editBookmark: (bookmark: IBookmark) => void;
  moveBookmark: (bookmarkId: string, newIndex: number) => void;
  removeBookmark: (bookmarkId: string) => void;
};

export const useSettingsUtilities: UseSettingsUtilities = () => {
...
```

Reply:
If it was input parameters it would be inside the brackets

Reply:
I mean that the name `props` means the input parameters of the function or component, but not the value that the function returns. Currently, it's a bit confusing